### ISSUE TITLE
Not defined link to 'git_blob_create_fromchunks'

### ISSUE THREAD

**Issue by lafisrap** [libgit2/git2go#306]
When I install git2go with 

> go get github.com/libgit2/git2go

Then I get following error:

> /tmp/go-build323687976/github.com/libgit2/git2go/_obj/wrapper.o: In Funktion `_go_git_blob_create_fromchunks':
> ../../../libgit2/git2go/wrapper.c:121: Nicht definierter Verweis auf`git_blob_create_fromchunks'
> collect2: error: ld returned 1 exit status

"Nicht definierter Verweis auf" means "Not defined link to"

How can I fix this?

**Comment by carlosmn** [comment]
You've installed an unreleased version of libgit2. Install the version which works with the version of git2go you have. See https://github.com/libgit2/git2go#stable-version

**Comment by lafisrap** [comment]
Yes, that's right. The README says that I need v0.22. At https://github.com/libgit2/libgit2/tags?after=v0.23.0-rc1 I found serveral 0.22s, and I tried 0.22.0, 0.22.2 and 0.22.3. I get different error messages from everyone. 

If I try to install git2go with go get github.com/gitlib2/git2go then I get:

could not determine kind of name for C.GIT_CHECKOUT_DONT_WRITE_INDEX
could not determine kind of name for C.GIT_CHECKOUT_RECREATE_MISSING

Where do I get the right version?
